github-extra/resources/gitconfig

```diff
@@ -32,10 +32,7 @@
 	format = html
 
 [http]
-	sslcainfo = /bin/curl-ca-bundle-ghfw.crt
-
-[sendemail]
-	smtpserver = /bin/msmtp.exe
+	sslcainfo = /ming32/usr/ssl/ca-bundle-ghfw.crt
```

Comment:
Yeah, I think you need to omit the `/ming32` part there. I noticed that it's omitted in the original git/git-for-windows code.